tests/language/comments.liq

```diff
@@ -0,0 +1,70 @@
+#<------#
+This is
+a multiline comment
+with > some # of
+these characters in it.
+#------>#
+
+#<---------------
+# Yet another
+# form of multiline comment
+ ---------------->#
+
+   #<
+multiline comment can start and end
+in the middle of a line
+      >#
+
+#<-----#
+Anything following them
+is executed
+#---->#x = 1
+
+#<-----#y = x + 2.
+Anything after the mark
+is comment
+#---->#
+
+#<---------------
+# Nested comments
+
+   #<
+     Are accepted
+   >#
+
+#---------------->#
+
+s = "foo
+#<-----
+this is a comment inside a string,
+it is not a code comment!
+#---->
+bar
+"
+ignore(s)
+
+r = r/foo
+#<-----
+this is a comment inside a regexp,
+it is not a code comment!
+#---->
+bar/
+ignore(r)
```

Comment:
`#---->`
This will trigger not terminated comment error?


I'd like to see these tests also
- `"a#b"` should be literally `a#b`;
- `r/a#b/i` should create a regex `r/a#b/i`;
- `"a#<b>#c"` should be literally `a#<b>#c`;
- `r/a#<b>#c/i` should create a regex `r/a#<b>#c/i`.